tools/validators/instance_validator/validate/subscriber.py

```diff
@@ -57,19 +57,19 @@ def Listen(self, callback):
       callback: a callback function to handle the message.
     """
     if self.service_account_info_json_file:
-      with open(self.service_account_info_json_file, encoding="utf-8") as f:
+      with open(self.service_account_info_json_file, encoding='utf-8') as f:
         service_account_info = json.load(f)
-      audience = "https://pubsub.googleapis.com/google.pubsub.v1.Subscriber"
+      audience = 'https://pubsub.googleapis.com/google.pubsub.v1.Subscriber'
       credentials = auth.jwt.Credentials.from_service_account_info(
           service_account_info, audience=audience)
     else:
-      print("No service account. Using application default credentials")
+      print('[INFO]\tNo service account. Using application default credentials')
       # pylint: disable=unused-variable
       credentials, project_id = auth.default()
 
     sub_client = pubsub_v1.SubscriberClient(credentials=credentials)
     future = sub_client.subscribe(self.subscription_name, callback)
-    print("Listening to pubsub, please wait ...")
+    print('[INFO]\tListening to pub/sub topic.')
```

Comment:
nit: keep the `please wait ...` as sometimes it takes time to read those messages. Otherwise the users will be impatient ?